src/test/java/test/cli/github1517/ExitCodeListenerBehaviorTest.java

```diff
@@ -18,9 +18,9 @@ public void testMethod(Class<?> clazz, int expectedStatus) {
     @DataProvider
     public Object[][] getData() {
         return new Object[][]{
-                {TestClassWithConfigFailureSample.class, 3},
-                {TestClassWithConfigSkipSample.class, 2},
-                {TestClassWithConfigSkipAndFailureSample.class, 3},
+                {TestClassWithConfigFailureSample.class, 6},
```

Comment:
Why do you need to change the values?

`TestClassWithConfigSkipAndFailureSample` was 3 => `HAS_SKIPPED + HAS_FAILURE` and is now 6 => `HAS_SKIPPED + HAS_FSP`. Is it expected?

Reply:
The logic is now different. The below table should summarize it.

| Skipped Bit | Failed Bit | Success Bit | Actual Value | Remarks                          |
|-------------|------------|-------------|--------------|----------------------------------|
| 0           | 0          | 1           | 1            | Test contains only success tests |
| 0           | 1          | 0           | 2            | Test contains only failures      |
| 1           | 0          | 0           | 4            | Only skipped tests               |
| 0           | 1          | 1           | 3            | Passed + Failed tests            |
| 1           | 0          | 1           | 5            | Passed + Skipped tests           |
| 1           | 1          | 1           | 7            | Passed + Failed + Skipped tests  |

Reply:
In that case, you assume the api changes and you need to remove current constants and modify `hasXxx` methods.

Reply:
Sure I will fix it.

Reply:
In fact, we can still keep the previous behavior if we remove the success bit.